src/nspawn/nspawn-mount.c

```diff
@@ -435,6 +433,11 @@ int mount_all(const char *dest,
                                 o = options;
                 }
 
+                if (streq_ptr(mount_table[k].where, "/sys/fs/selinux") && is_dir("/sys/fs/selinux", true) < 0) {
+                        log_debug("SELINUX not active, skipping selinux mounts.");
+                        continue;
+                }
```

Comment:
well, depends. I'd downgrade it to LOG_DEBUG if "fatal" is set to false for the entry. @keszybz opinions?